promo/js/main.js

```diff
@@ -15,6 +15,48 @@ jQuery(function() {
   //translate plugin
   //jQuery('#langPicker').translate();
 
+    //Vacatures link
+  var jobclassName;
+  var vacatureItem;
+
+  jQuery('.job1').click(function () {
+
+    jobclassName = jQuery(this).attr('class');
+    vacatureItem = '';
+
+    if (jobclassName == 'job1' && vacatureItem == undefined || vacatureItem == '') {
+
+      jQuery('#TextContainer1').show();
+      jQuery('#TextContainer2').hide();
+      return false;
+    }
+  });
+
+
+
+  jQuery('.job2').click(function () {
+
+    jobclassName = jQuery(this).attr('class');
+    vacatureItem = '';
+
+    if (jobclassName == 'job2' && vacatureItem == undefined || vacatureItem == '') {
+      jQuery('#TextContainer1').hide();
```

Comment:
I think "TextContainer1" is a bad name. Also "TextContainer2" is not very desriptive